test/quantization/core/test_quantized_op.py

```diff
@@ -2344,7 +2344,7 @@ def test_group_norm(self):
                         ch_end = ch_start + channels_per_group
                         group_vals = dqX[batch_idx][ch_start:ch_end]
                         assume(
-                            float(torch.unique(group_vals).shape[0]) / group_vals.numel() > 0.01
+                            float(torch.unique(group_vals).shape[0]) / group_vals.numel() > 0.001
```

Comment:
Hi @vkuzo. Thanks for reviewing.
It is because after changing input dimensions, the assume statement will fail.
```diff
-        num_groups_list = (1, 2)
-        channels_per_groups = (1, 2)
-        elements_per_channels = (8, 17)
+        num_groups_list = (1, 4)
+        channels_per_groups = (1, 36, 72)
+        elements_per_channels = (8, 128, 1024)
```
Is it OK to do so?